### ISSUE TITLE
Add accessibility section

### ISSUE THREAD

**Issue by LJWatson** [w3c/web-share#145]
In the [WebApps charter](https://www.w3.org/2019/05/webapps-charter.html#scope) it says:
>Where there are implications for implementors, developers, or users, in the areas of accessibility, internationalization, privacy, and security, each specification must have a section that describes relevant benefits, limitations, and best practice solutions for that particular area.>

To this end each spec needs to have an accessibility section. For the avoidance of doubt in the reader, the section should be included even if it indicates there are no known accessibility considerations.

Relates to w3c/webappswg#15

**Comment by ewilligers** [comment]
We could add a sentence like

> When the user agent on a given platform presents the user with a choice of target applications, the UI should support the assistive technologies commonly used on the platform.

**Comment by marcoscaceres** [comment]
Something like that would be good. I guess we should make it clear that it's not usually the browser that's is in control of the the share UI. However, in cases where it is, best effort should be made to adhere to OS accessibility guidelines that work best with assistive technologies. 

@LJWatson, wdyt? could you provide us with a bit more guidance on how other specs have handled this kind of situation (i.e., where the UI is often outside the control of the spec)?

**Comment by LJWatson** [comment]
The [Framework for Accessible Specification of Technologies (FAST)](https://w3c.github.io/apa/fast/) and the associated [FAST checklist](https://w3c.github.io/apa/fast/checklist.html) might be helpful, but I think you're on the right lines.

I'd suggest something like the following:
>When this specification is used to present information in the user interface it should conform to the [Web Content Accessibility Guidelines](https://w3.org/tr/wcag/) or follow OS level accessibility guidelines for the platform.>

You might want to expand it a bit to make it clear that WCAG is for content in the browser and OS level guidelines for information presented as part of an application or at the OS level itself though.

**Comment by LJWatson** [comment]
Talking this through with @marcoscaceres I realise that authors have no particular influence over the way this specification is used to present information in the user interface, so the accessibility section just needs to focus on considerations for implementors. Suggest:

>When this specification is used to present information in the user interface implementors should follow the OS level accessibility guidelines for the platform.>
